{"uid":"6565f437ca1bcda","name":"positive_check_discount_displayed_for_unauthorized_customer_for_amount_15000rub_and_above_smoke","fullName":"tests.test_products_on_sale.test_products_on_sale_page.TestProductsOnSalePage#test_positive_check_discount_for_unauthorized_customer_for_amount_15000rub_and_above_smoke","historyId":"6933e1382b5c2e443bb13025031c44fc","time":{"start":1717117811392,"stop":1717117813371,"duration":1979},"status":"skipped","statusMessage":"XFAIL Ждем исправление бага - Разная цена товара в каталоге и на карточке товара\n\nselenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame\n  (Session info: chrome-headless-shell=125.0.6422.141); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nStacktrace:\n#0 0x5563b379be3a <unknown>\n#1 0x5563b348545c <unknown>\n#2 0x5563b348ab69 <unknown>\n#3 0x5563b348c915 <unknown>\n#4 0x5563b348c99c <unknown>\n#5 0x5563b34d824c <unknown>\n#6 0x5563b34d6176 <unknown>\n#7 0x5563b34d3b57 <unknown>\n#8 0x5563b34d297c <unknown>\n#9 0x5563b34c60ff <unknown>\n#10 0x5563b34f44b2 <unknown>\n#11 0x5563b34c5a79 <unknown>\n#12 0x5563b34f467e <unknown>\n#13 0x5563b35132cc <unknown>\n#14 0x5563b34f4253 <unknown>\n#15 0x5563b34c41c7 <unknown>\n#16 0x5563b34c4b3e <unknown>\n#17 0x5563b376227b <unknown>\n#18 0x5563b3766327 <unknown>\n#19 0x5563b374edae <unknown>\n#20 0x5563b3766df2 <unknown>\n#21 0x5563b373374f <unknown>\n#22 0x5563b378b128 <unknown>\n#23 0x5563b378b2fb <unknown>\n#24 0x5563b379af6c <unknown>\n#25 0x7fae09498134 <unknown>","statusTrace":"self = <tests.test_products_on_sale.test_products_on_sale_page.TestProductsOnSalePage object at 0x7fde524a6b50>\nlink = 'https://test2.stroyrem-nn.ru/catalog/rastvoronasosy'\nproduct_page_open = <pages.products_on_sale_page.ProductsOnSale object at 0x7fde521d74d0>\n\n    @allure.title(\"positive_check_discount_displayed_for_unauthorized_customer_for_amount_15000rub_and_above_smoke\")\n    @pytest.mark.parametrize('link', [f\"{MAIN_PAGE_PROD_URL}{RASTVORONASOSY_PAGE_URL}\",\n                                      f\"{MAIN_PAGE_STAGE_URL}{RASTVORONASOSY_PAGE_URL}\"])\n    @pytest.mark.xfail(reason=\"Ждем исправление бага - Разная цена товара в каталоге и на карточке товара\")\n    @pytest.mark.smoke_test\n    def test_positive_check_discount_for_unauthorized_customer_for_amount_15000rub_and_above_smoke(self, link,\n                                                                                                   product_page_open):\n        product_page_open.get_field_price_first().send_keys(15000)\n        product_page_open.get_field_price_last().click()\n        time.sleep(1)\n>       product_page_open.get_add_to_cart_btn().click()\n\ntests/test_products_on_sale/test_products_on_sale_page.py:69: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/usr/local/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py:94: in click\n    self._execute(Command.CLICK_ELEMENT)\n/usr/local/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute\n    return self._parent.execute(command, params)\n/usr/local/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fde5243a290>\nresponse = {'status': 404, 'value': '{\"value\":{\"error\":\"stale element reference\",\"message\":\"stale element reference: stale elemen...\\\\n#23 0x5563b378b2fb \\\\u003Cunknown>\\\\n#24 0x5563b379af6c \\\\u003Cunknown>\\\\n#25 0x7fae09498134 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame\nE         (Session info: chrome-headless-shell=125.0.6422.141); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nE       Stacktrace:\nE       #0 0x5563b379be3a <unknown>\nE       #1 0x5563b348545c <unknown>\nE       #2 0x5563b348ab69 <unknown>\nE       #3 0x5563b348c915 <unknown>\nE       #4 0x5563b348c99c <unknown>\nE       #5 0x5563b34d824c <unknown>\nE       #6 0x5563b34d6176 <unknown>\nE       #7 0x5563b34d3b57 <unknown>\nE       #8 0x5563b34d297c <unknown>\nE       #9 0x5563b34c60ff <unknown>\nE       #10 0x5563b34f44b2 <unknown>\nE       #11 0x5563b34c5a79 <unknown>\nE       #12 0x5563b34f467e <unknown>\nE       #13 0x5563b35132cc <unknown>\nE       #14 0x5563b34f4253 <unknown>\nE       #15 0x5563b34c41c7 <unknown>\nE       #16 0x5563b34c4b3e <unknown>\nE       #17 0x5563b376227b <unknown>\nE       #18 0x5563b3766327 <unknown>\nE       #19 0x5563b374edae <unknown>\nE       #20 0x5563b3766df2 <unknown>\nE       #21 0x5563b373374f <unknown>\nE       #22 0x5563b378b128 <unknown>\nE       #23 0x5563b378b2fb <unknown>\nE       #24 0x5563b379af6c <unknown>\nE       #25 0x7fae09498134 <unknown>\n\n/usr/local/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:229: StaleElementReferenceException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1717116834579,"stop":1717116834656,"duration":77},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"clear_allure_results_folder","time":{"start":1717116834656,"stop":1717116834656,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver","time":{"start":1717117797197,"stop":1717117797884,"duration":687},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup","time":{"start":1717117797884,"stop":1717117797884,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"product_page_open","time":{"start":1717117797884,"stop":1717117811392,"duration":13508},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"skipped","statusMessage":"XFAIL Ждем исправление бага - Разная цена товара в каталоге и на карточке товара\n\nselenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame\n  (Session info: chrome-headless-shell=125.0.6422.141); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nStacktrace:\n#0 0x5563b379be3a <unknown>\n#1 0x5563b348545c <unknown>\n#2 0x5563b348ab69 <unknown>\n#3 0x5563b348c915 <unknown>\n#4 0x5563b348c99c <unknown>\n#5 0x5563b34d824c <unknown>\n#6 0x5563b34d6176 <unknown>\n#7 0x5563b34d3b57 <unknown>\n#8 0x5563b34d297c <unknown>\n#9 0x5563b34c60ff <unknown>\n#10 0x5563b34f44b2 <unknown>\n#11 0x5563b34c5a79 <unknown>\n#12 0x5563b34f467e <unknown>\n#13 0x5563b35132cc <unknown>\n#14 0x5563b34f4253 <unknown>\n#15 0x5563b34c41c7 <unknown>\n#16 0x5563b34c4b3e <unknown>\n#17 0x5563b376227b <unknown>\n#18 0x5563b3766327 <unknown>\n#19 0x5563b374edae <unknown>\n#20 0x5563b3766df2 <unknown>\n#21 0x5563b373374f <unknown>\n#22 0x5563b378b128 <unknown>\n#23 0x5563b378b2fb <unknown>\n#24 0x5563b379af6c <unknown>\n#25 0x7fae09498134 <unknown>","statusTrace":"self = <tests.test_products_on_sale.test_products_on_sale_page.TestProductsOnSalePage object at 0x7fde524a6b50>\nlink = 'https://test2.stroyrem-nn.ru/catalog/rastvoronasosy'\nproduct_page_open = <pages.products_on_sale_page.ProductsOnSale object at 0x7fde521d74d0>\n\n    @allure.title(\"positive_check_discount_displayed_for_unauthorized_customer_for_amount_15000rub_and_above_smoke\")\n    @pytest.mark.parametrize('link', [f\"{MAIN_PAGE_PROD_URL}{RASTVORONASOSY_PAGE_URL}\",\n                                      f\"{MAIN_PAGE_STAGE_URL}{RASTVORONASOSY_PAGE_URL}\"])\n    @pytest.mark.xfail(reason=\"Ждем исправление бага - Разная цена товара в каталоге и на карточке товара\")\n    @pytest.mark.smoke_test\n    def test_positive_check_discount_for_unauthorized_customer_for_amount_15000rub_and_above_smoke(self, link,\n                                                                                                   product_page_open):\n        product_page_open.get_field_price_first().send_keys(15000)\n        product_page_open.get_field_price_last().click()\n        time.sleep(1)\n>       product_page_open.get_add_to_cart_btn().click()\n\ntests/test_products_on_sale/test_products_on_sale_page.py:69: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/usr/local/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py:94: in click\n    self._execute(Command.CLICK_ELEMENT)\n/usr/local/lib/python3.11/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute\n    return self._parent.execute(command, params)\n/usr/local/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fde5243a290>\nresponse = {'status': 404, 'value': '{\"value\":{\"error\":\"stale element reference\",\"message\":\"stale element reference: stale elemen...\\\\n#23 0x5563b378b2fb \\\\u003Cunknown>\\\\n#24 0x5563b379af6c \\\\u003Cunknown>\\\\n#25 0x7fae09498134 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame\nE         (Session info: chrome-headless-shell=125.0.6422.141); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nE       Stacktrace:\nE       #0 0x5563b379be3a <unknown>\nE       #1 0x5563b348545c <unknown>\nE       #2 0x5563b348ab69 <unknown>\nE       #3 0x5563b348c915 <unknown>\nE       #4 0x5563b348c99c <unknown>\nE       #5 0x5563b34d824c <unknown>\nE       #6 0x5563b34d6176 <unknown>\nE       #7 0x5563b34d3b57 <unknown>\nE       #8 0x5563b34d297c <unknown>\nE       #9 0x5563b34c60ff <unknown>\nE       #10 0x5563b34f44b2 <unknown>\nE       #11 0x5563b34c5a79 <unknown>\nE       #12 0x5563b34f467e <unknown>\nE       #13 0x5563b35132cc <unknown>\nE       #14 0x5563b34f4253 <unknown>\nE       #15 0x5563b34c41c7 <unknown>\nE       #16 0x5563b34c4b3e <unknown>\nE       #17 0x5563b376227b <unknown>\nE       #18 0x5563b3766327 <unknown>\nE       #19 0x5563b374edae <unknown>\nE       #20 0x5563b3766df2 <unknown>\nE       #21 0x5563b373374f <unknown>\nE       #22 0x5563b378b128 <unknown>\nE       #23 0x5563b378b2fb <unknown>\nE       #24 0x5563b379af6c <unknown>\nE       #25 0x7fae09498134 <unknown>\n\n/usr/local/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:229: StaleElementReferenceException","steps":[{"name":"Проверяем видимость поля 'ЦЕНА от'","time":{"start":1717117811392,"stop":1717117811426,"duration":34},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Проверяем видимость поля 'ЦЕНА до'","time":{"start":1717117811472,"stop":1717117811500,"duration":28},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Проверяем кликабельность кнопки 'В корзину'","time":{"start":1717117812534,"stop":1717117812581,"duration":47},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[{"uid":"780165fc714667a","name":"Screenshot","source":"780165fc714667a.png","type":"image/png","size":59578},{"uid":"718e5641aed60356","name":"HTML source","source":"718e5641aed60356.html","type":"text/html","size":520212}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":3,"attachmentsCount":2,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"setup::0","time":{"start":1717117813495,"stop":1717117813495,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver::0","time":{"start":1717117813495,"stop":1717117813571,"duration":76},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"epic","value":"Products On SalePage"},{"name":"tag","value":"smoke_test"},{"name":"tag","value":"@pytest.mark.xfail(reason='Ждем исправление бага - Разная цена товара в каталоге и на карточке товара')"},{"name":"tag","value":"flaky(max_runs=3, min_passes=1)"},{"name":"parentSuite","value":"tests.test_products_on_sale"},{"name":"suite","value":"test_products_on_sale_page"},{"name":"subSuite","value":"TestProductsOnSalePage"},{"name":"host","value":"373c34aee6e9"},{"name":"thread","value":"1-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_products_on_sale.test_products_on_sale_page"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"link","value":"'https://test2.stroyrem-nn.ru/catalog/rastvoronasosy'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":2,"passed":0,"unknown":0,"total":2},"items":[{"uid":"4f0b266c25e13e3e","reportUrl":"https://Nat754.github.io/StroyremAutomation/323//#testresult/4f0b266c25e13e3e","status":"skipped","statusDetails":"XFAIL Ждем исправление бага - Разная цена товара в каталоге и на карточке товара\n\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5643419b9e8a <unknown>\n#1 0x5643416a345c <unknown>\n#2 0x5643416ef5b5 <unknown>\n#3 0x5643416ef671 <unknown>\n#4 0x564341733f14 <unknown>\n#5 0x5643417124dd <unknown>\n#6 0x5643417312cc <unknown>\n#7 0x564341712253 <unknown>\n#8 0x5643416e21c7 <unknown>\n#9 0x5643416e2b3e <unknown>\n#10 0x5643419802cb <unknown>\n#11 0x564341984377 <unknown>\n#12 0x56434196cdfe <unknown>\n#13 0x564341984e42 <unknown>\n#14 0x56434195179f <unknown>\n#15 0x5643419a9178 <unknown>\n#16 0x5643419a934b <unknown>\n#17 0x5643419b8fbc <unknown>\n#18 0x7faa4f012134 <unknown>","time":{"start":1717031145293,"stop":1717031161598,"duration":16305}}]},"tags":["smoke_test","@pytest.mark.xfail(reason='Ждем исправление бага - Разная цена товара в каталоге и на карточке товара')","flaky(max_runs=3, min_passes=1)"]},"source":"6565f437ca1bcda.json","parameterValues":["'https://test2.stroyrem-nn.ru/catalog/rastvoronasosy'"]}